var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -131,10 +131,11 @@ def cmake_args(self):
             )
         return args
 
-    def test(self):
+    def test_drivers(self):
+        """Test drivers"""
         if self.spec.satisfies("@:5.5.0"):
-            print("Skipping: stand-alone tests")
-            return
+            raise SkipTest("Package must be installed as version @5.5.1 or later")
         test_dir = join_path(self.spec["migraphx"].prefix, "bin")
         with working_dir(test_dir, create=True):
```

Comment:
This is running from within the install prefix, which it is not supposed to do.

Reply:
Also, that directory should already be there if the software was successfully installed (so `create=True` is irrelevant).

Reply:
Is it not possible to run using the Executable form of `self.spec["migraphx"].prefix.bin.UnitTests`?